doc/chapters/filesystem/index.rst

```diff
@@ -136,3 +136,36 @@ changes performed during its runtime to be merged into the lower layers.
 .. Note:: Non-directory types of files can not be mounted using overlayfs.
           These will automatically fall back on using the default behavior of 
           bind mounting the files into the filesystem of the container.
+
+Temporary Filesystem
+====================
+
+The Write buffers can be configured to use a separate temporary filesystem
+(``tmpfs``) which can be limited in size. The ``tmpfs`` is mounted on top of
+the containers temporary directory as soon as it's created and will remain
+there until the container is destroyed. All ``overlayfs`` mounts and temporary
+directories should be created inside this ``tmpfs`` mount.
+
+The ``size`` of the ``tmpfs`` can also be limited using an extra configuration
+option in the ``Create DBus`` call to the ``SoftwareContainerAgent``.
```

Comment:
Should "Create DBus" really by code highlighted? I guess you only want the method name to be that?